src/Microsoft.OpenApi/Expressions/RuntimeExpression.cs

```diff
@@ -42,19 +42,19 @@ public static RuntimeExpression Build(string expression)
             // $url
             if (expression == UrlExpression.Url)
             {
-                return new UrlExpression();
+                return UrlExpression.Instance;
```

Comment:
>UrlExpression.Instance; [](start = 23, length = 23)

I can see the benefits of the singleton pattern here, but I think it can make things a bit confusing. (Using .Instance is less natural then new Class())

Reply:
I don't think there's any benefits to use new Class().  I mean for "$method, $url, $statusCode", they are no other information. It's be better to use singleton.

Reply:
That's fine. Let's roll with this.